au3/libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -220,10 +220,14 @@ double WaveClipChannel::GetStretchRatio() const
    return GetClip().GetStretchRatio();
 }
 
+
+static int64_t s_lastClipId = 0;
+
 WaveClip::WaveClip(size_t width,
```

Comment:
WaveClip::WaveClip(size_t width, const SampleBlockFactoryPtr &factory, sampleFormat format, int rate) 
    : mId(++s_lastClipId), mRate(rate), mSequences(width) {
    assert(width > 0);
}
In the constructor, initialize member variables directly in the initializer list rather than in the body. For instance, mId, mRate, and mSequences.resize(width) should ideally be in the initializer list:

Reply:
made in accordance with the current AU3 style